akka-docs-new/src/main/paradox/guide/actors-intro.md

```diff
@@ -0,0 +1,259 @@
+# What problems does the actor model solve? 
+
+Akka uses the actor model to overcome the limitations of traditional object-oriented programming models and meet the 
+unique challenges of highly distributed systems. To fully understand why the actor model is necessary, it helps to 
+identify mismatches between traditional approaches to programming and the realities of concurrent and distributed 
+computing. 
+
+### The illusion of encapsulation
+
+Object oriented programming (OOP) is a widely-accepted, familiar programming model. One of its basic pillars is 
+_encapsulation_. Encapsulation dictates that the internal data of an object is not accessible directly from the outside; 
+it can only modified by invoking a set of curated methods. The object is responsible to expose only safe operations 
+that protect the invariant nature of its encapsulated data.
+
+For example, operations on an ordered binary tree implementation must not allow violation of the tree ordering 
+constraint. Callers rely on the ordering being intact. For example, when querying the tree for a certain piece of 
```

Comment:
technically both are correct, but you are right, I will just use invariant, no need for confusion.